rest_framework_simplejwt/settings.py

```diff
@@ -24,6 +24,7 @@
     'AUTH_HEADER_TYPES': ('Bearer',),
     'USER_ID_FIELD': 'id',
     'USER_ID_CLAIM': 'user_id',
+    'USER_AUTHENTICATION_RULE': 'rest_framework_simplejwt.authentication.default_user_authentication_rule',
```

Comment:
@Andrew-Chen-Wang @t-io 

If we add this setting name to `rest_framework_simplejwt.settings.IMPORT_STRINGS`, we won't need to do the string splitting and importlib stuff in `rest_framework_simplejwt/serializers.py`.